packages/jest-cli/src/lib/PromptController.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const {KEYS} = require('../constants');
+
+class PromptController {
+  entering: boolean;
```

Comment:
Let's call this "_isEntering". Please add a `_` to all fields that are supposed to be private.